.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
If you can remove this file from the PR it would be great. 
We haven't decided on config files for editors yet.